src/Microsoft.AspNetCore.SignalR/GetHttpContextExtensions.cs

```diff
@@ -1,16 +1,21 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Http.Connections.Features;
 
 namespace Microsoft.AspNetCore.SignalR
 {
-    public static class HubCallerContextExtensions
+    public static class GetHttpContextExtensions
     {
         public static HttpContext GetHttpContext(this HubCallerContext connection)
         {
             return connection.Features.Get<IHttpContextFeature>()?.HttpContext;
         }
+
+        public static HttpContext GetHttpContext(this HubConnectionContext connection)
+        {
+            return connection.Features.Get<IHttpContextFeature>()?.HttpContext;
```

Comment:
Check `connection` argument is not null? +other overload

Reply:
Sure, but it should `ArgumentNullException` there. I'll add that.